src/LMTR_alg.jl

```diff
@@ -130,7 +130,8 @@ function LMTR(
   JdFk = similar(Fk)   # temporary storage
   Jt_Fk = similar(∇fk)   # temporary storage
 
-  σmax = opnorm(Jk)
+  σmax, found_σ = opnorm(Jk)
+  found_σ || error("operator norm computation failed")
```

Comment:
We should have a backup plan if the computation fails.

Reply:
Working on understanding why it may not work ^^

Reply:
Done